### DISCUSSION TITLE
使用tcp转https方式是否无法加载本地证书？

### DISCUSSION THREAD

**Discussion by john41nt** [natfrp/wiki#505] [使用问题]
之前使用海外节点，自己的域名aaa.bbb.com，在本地工作目录配置好证书aaa.bbb.com.crt和aaa.bbb.com.key，能够直接访问https://aaa.bbb.com 打开网站，证书没问题

今天换国内tcp节点济南联通plus，分配端口56789，访问https://aaa.bbb.com:56789 可以打开网站，但是证书还是sakura自签发，未能加载本地证书，请问是否为预期特性？
如果这种访问方式不合法也请告知

**Comment by vfr456** [comment]
好像国内的节点没备案的话没办法用https， 非443端口也不行。

在网上搜到过说用ip+非443端口可以不备案使用https服务，不知道natfrp里行不行，谁知道麻烦告知一下。

**Reply by vfr456** [reply]
更正：国内节点tcp隧道，非标准端口https，是可以用的。  具体配置为：本地机器上配置自有域名的证书，DNS CNAME解析到节点域名，然后使用 https://{自有域名}:{隧道端口} 就可以了。   不知道后续会不会被拦截

**Comment by fakeboboliu** [comment]
请将 tcp 隧道的 自动HTTPS (`auto_https`) 项目内容设为你的 aaa.bbb.com 即可

**Reply by john41nt** [reply]
看了下国内节点不允许这么设置，那我知道能不能这么用了，谢谢
